include/aws/checksums/exports.h

```diff
@@ -4,19 +4,34 @@
  * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
  * SPDX-License-Identifier: Apache-2.0.
  */
-#if defined(USE_WINDOWS_DLL_SEMANTICS) || defined(WIN32)
-
-#    ifdef USE_IMPORT_EXPORT
+#if defined(AWS_C_RT_USE_WINDOWS_DLL_SEMANTICS) || defined(_WIN32)
+#    ifdef AWS_COMMON_USE_IMPORT_EXPORT
 #        ifdef AWS_CHECKSUMS_EXPORTS
 #            define AWS_CHECKSUMS_API __declspec(dllexport)
 #        else
 #            define AWS_CHECKSUMS_API __declspec(dllimport)
 #        endif /* AWS_CHECKSUMS_EXPORTS */
 #    else
 #        define AWS_CHECKSUMS_API
-#    endif /* USE_IMPORT_EXPORT */
-#else      /* defined (USE_WINDOWS_DLL_SEMANTICS) || defined (WIN32) */
-#    define AWS_CHECKSUMS_API
-#endif /* defined (USE_WINDOWS_DLL_SEMANTICS) || defined (WIN32) */
+#    endif /* AWS_COMMON_USE_IMPORT_EXPORT */
```

Comment:
copy/paste error.

Reply:
Looks like if I use `USE_IMPORT_EXPORT`, it still have error.

Reply:
? You coppied AWS_COMMON_* stuff

It should be AWS_CHECKSUMS right?